internal/template-validator/tlsinfo/tlsinfo.go

```diff
@@ -192,8 +193,8 @@ func (ti *TLSInfo) CreateTlsConfig() *tls.Config {
 	}
 
 	if !ti.sspTLSOptions.IsEmpty() {
-		tlsConfig.CipherSuites = ti.sspTLSOptions.CipherIDs(nil)
-		minVersion, err := ti.sspTLSOptions.MinTLSVersionId()
+		tlsConfig.CipherSuites = common.CipherIDs(ti.sspTLSOptions.OpenSSLCipherNames)
```

Comment:
Is there a lib for this too?

Reply:
There is this function, but the problem is that it causes panic for some cipher names and we just want  to log that the cipher is not supported  https://github.com/kubevirt/ssp-operator/blob/fc1d67c1b3ee193d02cab94149217525975c1f0c/vendor/github.com/openshift/library-go/pkg/crypto/crypto.go#L233-L245